content/_releases/22/34.md

```diff
@@ -0,0 +1,171 @@
+  - /releases/22.34.html
+---
+
+## Improvements and Updates
+
+### Meaningful error for invalid SSH key entry
+
+When we introduced [support for newest SSH versions](/releases/22/14#ssh-keys-and-ssh-88)
+for the user SSH keys used to deploy components, we deprecated the old `ssh-dss`
+types. Now the user-interface shows a distinct error explaining the situation if
+the user enters an un-supported or malformed SSH Key instead of just disabling
+the Save button like it was before.
+
+{% include img.html max-width="80%" url="/assets/img/RN/2234/RN2234-ssh-key-screen-error.png" title="SSH Key screen in case of Error" %}
+
+### Deployment of Externally Build Dockers
+
+Our team has just begun working on a new exciting feature to enable deployment of
+externally build docker images for the integration components. This feature might
+replace or provide an alternative to git push deployment of user developed components.
+
+At this earliest stages we lay our groundwork by updating the component
+`commons` and `api` to support components pull from external docker registries.
+The following new attributes were added:
+
+*   `docker_registry` object into the an HTTP `POST` call to `/v2/teams/`
+    *   `attributes.docker_registry.uri` - Docker Registry URI
+    *   `attributes.docker_registry.credentials` - Docker Registry credentials
+*   `docker_repo_name` and `docker_target_tag` in the `POST` HTTP call to `/v2/components` (can be set if its team has `docker_registry` object)
+    *   `attributes.docker_repo_name` - String representing component's name in docker registry. Available only if team supports docker registry
+    *   `attributes.docker_target_tag` - String representing component's tag in docker registry. Available only if team supports docker registry
+
+> Please note, as of this stage these attributes have no functionality attached to them.
+
+{% if site.data.tenant.name == "elastic.io" %}
+
+## HELM3
+
+We constantly improve different aspects of HELM3 deployments in collaboration
+with our partners. This section lists updates and improvements included in this release.
+
+### Dashboard Runlog retention period
+
+The flow execution statistics presented on the Dashboard Runlog is stored
+in the platform Mongo Database. This is in contrast to the flow execution threads, shown
+on the Executions pages, which is stored separately in the Clickhouse.
+
+When you access the Dashboard Runlog, the platform queries the MongoDB to get these
+records. This call to the Database can slow-down the page load for the high-end users.
+For this purposes it is prudent to limit the retention period of data in the Database.
+
+Previously you could only control the retention period by enforcing `expireAfterSeconds`
+option in the Database index holding the execution records. In this release we
+introduce a new `TASK_STAT_START_INDEX_TTL` environment variable to the HELM3 charts,
+which you can use to set retention suitable for your environment. If not specified,
```

Comment:
environment. It is mandatory that a value is set for `TASK_STAT_START_INDEX_TTL` and the default value is set at 432000....

**This means that** the platform will...
Delete 'only'

Reply:
done as you've suggested